README.md

```diff
@@ -1,7 +1,7 @@
 # angular-bind-html-compile
 This repo contains a bower package that provides an angular directive which can be passed trusted html with angular template content to evaluate.
 
-The `bind-html-compile` directive allows for HTML containing directives to be compiled.
+The `bind-html-compile` directive allows for HTML containing **custom** directives to be compiled.
```

Comment:
I don't think that this change is needed, since it doesn't matter if these are custom or angular core directives